docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+Advanced Options
+""""""""""""""
+More advanced users have the possibility to manually set all of the actual CCDC parameters by clicking on the :guilabal:`MORE` button.
+
+.. thumbnail:: ../_images/cookbook/ccdc_asset/opt_ccdc_advanced.png
+    :title: Opt Selection - Advanced
+    :width: 49%
+    :align: center
+    :group: ccdc-asset-recipe
+
+|
+**Date Format**
+
+This option allows to safe the dates in various formats. Note that SEPAL deals by default with :guilabel:`FRACTIONAL YEARS` in all of the CCDC related recipes.
+
+**TMASK BANDS**
+
+The bands selected here are used for additional multi-temporal filtering of cloud affected pixels that have not been identified as such throughout the pre-processing of the single images. For optical data from Landsat and Sentinel-2, the :guilabel:`GREEN` and :guilabel:`SWIR1` bands are recommended.
+
+**Min Observations**
+
+This is the number of observations needed before a break is actually confirmed based on its temporal behaviour. A low number will lead to more changes and reduce the gaps between two temporal segments. Higher numbers will lead to more confidence in the observed change, but in cloud-prone regions might lead to long gaps between two temporal segments. Usually, a number between 4 to 8 is recommended.
+
+**Chi Square Probability**
+
+The Chi-Square test will test if an observation is part of the general statistical distribution of the time-series. A low value of Chi-Square probability will favor the rejection of the null-hypothesis (i.e. being part of the statistical distribution), therefore flagging it as possible change. Ultimately, a lower value leads to more breaks detected, which favors omission over commission error.
+A high value allows for more noise in the time-series, and less changes will be detected, therefore lowering the commission error rate.
+
+**Min Number of Years Scaler**
+
+This parameter determines the minimum length of any inner temporal segment.
+
+**LAMBDA**
+
+The lambda parameter is part of the LASSO regression used for the modelling of the time-series. It is used to generalize the model and thereby improving its predictive power. More specifically, it is controlling the weight of each of the parameters, and might result even in the annulation of some of the parameters. In practical terms, an initially 3rd order harmonic model, might shrink to a 1st order harmonic, if this provides the best generalized fit. Setting lambda to 0 will lead to a regular Ordinary-Least-Square regression, not providing any generalization. Instead, a higher value will provide a more generalised model. If lambda is set too high, the model will underfit, which also not wanted. Since a value of 20 has been found to provide a generally good performance, the sweet spot of neither over- nor underfitting will be around this number.
```

Comment:
typo: "over- nor" ? for me there should not be a space but maybe that's my poor level of english

Reply:
advanced settings - advanced germanized English ? not the space, but maybe the dash should be removed a in this headline here https://www.business-standard.com/article/pti-stories/india-s-gdp-growth-neither-over-nor-underestimated-says-eco-survey-120013100913_1.html

Reply:
that's just a comment if you think it does not require changes simply mark it as solved, I'll be fooled